code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -114,6 +114,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	..()
 	flags |= NOREACT // so it doesn't react until you light it
 	create_reagents(chem_volume) // making the cigarette a chemical holder with a maximum volume of 15
+	reagents.add_reagent("nicotine", 15)

```

Comment:
Just increase the chem capacity.
Op 12-jan.-2015 14:54 schreef "Iamgoofball" <notifications@github.com>:

> In code/game/objects/items/weapons/cigs_lighters.dm
> <https://github.com/tgstation/-tg-station/pull/6137#discussion-diff-22790615>
> :
>
> > @@ -114,6 +114,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
> >  	..()
> >  	flags |= NOREACT // so it doesn't react until you light it
> >  	create_reagents(chem_volume) // making the cigarette a chemical holder with a maximum volume of 15
> > +	reagents.add_reagent("nicotine", 15)
>
> I'll swap it to 10 so you can still dunk them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/6137/files#r22790615>.
>